templates/ts/non-sso-tab/src/static/scripts/teamsapp.ts

```diff
@@ -1,3 +1,5 @@
+import * as microsoftTeams from "@microsoft/teams-js";
```

Comment:
Also update the source code as it's TypeScript?
For example, adding type for hubName.

Reply:
Sure.